spec/units/commands/internal/chdir_spec.rb

```diff
@@ -6,14 +6,14 @@
 
   describe '#execute' do
     it 'returns true for correct directories' do
-      env = stub(:[]= => true, puts_error: true)
+      env = double("Environment", :[]= => true, puts_error: true)
```

Comment:
No problem, I do it too. Just trying to keep the code as consistent as possible.